Mapping/Loader/LoaderChain.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Algolia\AlgoliaSearchBundle\Mapping\Loader;
+
+use Doctrine\ORM\EntityManager;
+
+final class LoaderChain implements LoaderInterface
+{
+    private $loaders;
+
+    public function getMetaData($entity, EntityManager $em)
+    {
+        foreach ($this->loaders as $loader) {
+            if (null !== $metadata = $loader->getMetaData($entity, $em)) {
```

Comment:
The annotation loader currently returns false when no data properties/methods have been mapped to Algolia. The loader should be updated to reflect this.